matchbox_socket/src/webrtc_socket/mod.rs

```diff
@@ -160,26 +161,32 @@ async fn message_loop<M: Messenger>(
 
         select! {
             _  = &mut timeout => {
-                requests_sender.unbounded_send(PeerRequest::KeepAlive).expect("send failed");
-                // UNWRAP: we will only ever get here if there already was a timeout
-                let interval = keep_alive_interval.unwrap();
-                timeout = Either::Left(Delay::new(interval)).fuse();
+                requests_sender.unbounded_send(PeerRequest::KeepAlive).map_err(TrySendError::into_send_error)?;
+                if let Some(interval) = keep_alive_interval {
+                    timeout = Either::Left(Delay::new(interval)).fuse();
+                }
```

Comment:
On second thought, I think this would be better
```suggestion
                if let Some(interval) = keep_alive_interval {
                    timeout = Either::Left(Delay::new(interval)).fuse();
                } else {
                    error!("no keep alive timeout, please file a bug");
                }
```

Also as a general-rule of thumb, if we `if let` things that will always be true, we should error if they're not, so we don't hide potential bugs.